lightbluetent/templates/home/home.html

```diff
@@ -0,0 +1,36 @@
+{% extends "base.html" %}
+
+{% block body %}
+
+<p>Hello, <b>{{ crsid }}</b>. Here are the stands for your societies.</p>
```

Comment:
We use "stall" and "stand" throughout. I think we should choose one and I am a fan of "stall" but ultimately it's the same thing.